### PR TITLE
Add a couple tests for errors

### PR DESCRIPTION
**Pull request by musm**

### CODE REVIEW

**Comment by timholy** [comment]
Can you provide a little more justification for these tests?

File: test/constructed_images.jl [hunk 1]
```diff
@@ -5,11 +5,21 @@ using Base.Test
...
+type TestType end
+
 @testset "IO" begin
     workdir = joinpath(tempdir(), "Images")
     isdir(workdir) && rm(workdir, recursive=true)
     mkdir(workdir)
 
+    a = rand(Bool,5,5,5,5)
+    fn = joinpath(workdir, "5by5.png")
+    @test_throws ErrorException ImageMagick.save(fn, a)
```
Comment (musm) [inline test/constructed_images.jl:17]: Test for line https://github.com/JuliaIO/ImageMagick.jl/blob/master/src/ImageMagick.jl#L151

File: test/constructed_images.jl [hunk 2]
```diff
@@ -5,11 +5,21 @@ using Base.Test
...
+
+    a = [TestType() TestType()]
+    fn = joinpath(workdir, "5by5.png")
+    @test_throws MethodError ImageMagick.save(fn, a)
```
Comment (musm) [inline test/constructed_images.jl:21]: Test for line 
https://github.com/JuliaIO/ImageMagick.jl/blob/master/src/ImageMagick.jl#L146

**Comment by musm** [comment]
Added which lines they test for. Super, super trivial PR ;)

**Comment by timholy** [comment]
Got it, thanks.

**Pull request merged**